src/tracing/errors.test.ts

```diff
@@ -54,9 +54,9 @@ describe('beforeSend', () => {
     expect(beforeSend(ERROR, { originalException })).toBe(ERROR)
   })
 
-  it('propagates user rejected request errors', () => {
+  it('filters user rejected request errors', () => {
```

Comment:
nit: add the TODO here as well so implementer also adds test to support graceful handling

Reply:
the TODO will basically cause them to change the test right, since the test will fail?